### ISSUE TITLE
WCAG 2.1 Accessibility

### ISSUE THREAD

**Issue by Pacoup** [MicrosoftDocs/powerbi-docs#2886]
Are reports published to web accessible / WCAG 2.1 compliant for web users?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: bc005634-24c6-15c4-4352-e1455d63f4f3
* Version Independent ID: 577c460a-08cc-892e-c5f9-e6106958f856
* Content: [Publish to web from Power BI - Power BI](https://docs.microsoft.com/en-us/power-bi/collaborate-share/service-publish-to-web)
* Content Source: [powerbi-docs/collaborate-share/service-publish-to-web.md](https://github.com/MicrosoftDocs/powerbi-docs/blob/master/powerbi-docs/collaborate-share/service-publish-to-web.md)
* Service: **powerbi**
* Sub-service: **pbi-collaborate-share**
* GitHub Login: @maggiesMSFT
* Microsoft Alias: **maggies**

**Comment by mihart** [comment]
Thanks for your comment, @Pacoup . This issue has been assigned to the doc author for review.

**Comment by Pacoup** [comment]
Hi, is it possible to get an answer to this question? We really need to know whether we can publish Power BI reports online as is or if we have to create an accessible alternative with another tool for our Government of Canada website, which is mandated to pass WCAG 2.1. Thanks.

**Comment by maggiesMSFT** [comment]
Hi, @Pacoup -- here is the article about creating accessible Power BI reports: https://docs.microsoft.com/en-us/power-bi/create-reports/desktop-accessibility-creating-reports
Hope that helps answer your question!

**Comment by Pacoup** [comment]
@maggiesMSFT This does not answer the question. That article is about creating accessible reports inside Power BI, but makes no mention of reports **published to the web**.

Specifically, the mention on [Overview of accessibility in Power BI](https://docs.microsoft.com/en-us/power-bi/create-reports/desktop-accessibility-overview) seems to imply that Power BI reports are only accessible if accessed within Power BI Desktop or Power BI Mobile:

> Power BI is committed to the following accessibility standards. The standards help ensure that your Power BI experiences are accessible to as many people as possible. When you build accessible reports or dashboards, that content is accessible for anyone who views them using Power BI Mobile.

My understanding is that Power BI Mobile is only available for reports published within an organization, not publicly available reports on the web.

We need to know whether the reports can also be made accessible and hopefully responsive when published to the web, because we're working on a report which we intend to make available to the public at large. [Publish to web from Power BI](https://docs.microsoft.com/en-us/power-bi/collaborate-share/service-publish-to-web) makes no mention of this and I cannot find any specific information regarding this capability.

**Comment by maggiesMSFT** [comment]
Oh! This article https://docs.microsoft.com/en-us/power-bi/create-reports/desktop-accessibility-creating-reports definitely applies to reports on the web, i.e., the Power BI service. 
I'll check to see if it also applies to Publish to Web reports.

**Comment by heidiv** [comment]
Hello - curious if there was an answer to this? Are reports published to web WCAG 2.1. compliant? Thanks!

**Comment by maggiesMSFT** [comment]
Hi, @heidiv -- The answer back in July 2021 was no, reports published to web were not WCAG 2.1. compliant, but there was movement on that front. So I'm checking again.

**Comment by carolynzinn** [comment]
@maggiesMSFT have you found anything out about the movement to make reports published to the web WCAG 2.1 compliant?

**Comment by maggiesMSFT** [comment]
@carolynzinn -- thanks for bringing this back to top of mind. I'm asking again.

**Comment by melioraaa** [comment]
@maggiesMSFT - keen on hearing if there's any development on this topic. Thanks!

**Comment by maggiesMSFT** [comment]
HI, @melioraaa -- I'm checking again.

**Comment by maggiesMSFT** [comment]
Hi @Pacoup @melioraaa @carolynzinn @heidiv -- here's the latest answer:
"We have made significant progress in accessibility efforts for Power BI in the past year. However, I would still recommend sharing Ben’s statement below: “Power BI does not fully support all applicable WCAG 2.1 standards for reports published to the web.  We continue to invest in improving the accessibility of Power BI, but Power BI is not currently an option if full conformance to WCAG 2.1 is a requirement.” 

"You may also provide accessibility conformance reports here that would be relevant to Publish to Web reports. Customers may refer to the ACR (accessibility conformance report) for any current issues here by searching Power BI:

[Accessibility Conformance Reports | Microsoft Accessibility](https://www.microsoft.com/en-us/accessibility/conformance-reports)
